charts/kubewarden-defaults/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: kubewarden-policy-server
+description: A Helm chart for deploying a Kubewarden PolicyServer and some default
+policies.
```

Comment:
I think the YAML is not valid because of the carriage return.

Aside from that, I would drop the part about "some default policies". We don't know if and when we're going to do it. Right now this is not done, hence we should not mention it to avoid confusion

Reply:
I've removed the text about policies.